apps/test/unit/applab/FirebaseStorageTest.js

```diff
@@ -0,0 +1,91 @@
+import { expect } from '../../util/configuredChai';
+import FirebaseStorage from '../../../src/applab/firebaseStorage';
+import { getDatabase, getFirebase } from '../../../src/applab/firebaseUtils';
+
+describe('FirebaseStorage', () => {
+  beforeEach(() => {
+    window.Applab = {
+      channelId: "test-firebase-channel-id",
+      firebaseName: 'test-firebase-name',
+      firebaseAuthToken: 'test-firebase-auth-token',
+    };
+  });
+
+  describe('populateTable', () => {
+    const EXISTING_TABLE_DATA = {
+      cities: {
+        records: {
+          1: '{"city":"New York","state":"NY","id":1}'
+        }
+      }
+    };
+    const NEW_TABLE_DATA_JSON =
+      '{' +
+      '  "cities": [' +
+      '    {"city": "Seattle", "state": "WA"},' +
+      '    {"city": "Chicago", "state": "IL"}' +
+      '  ]' +
+      '}';
```

Comment:
Assuming this isn't whitespace-sensitive, you could use a multiline template literal here:
```javascript
    const NEW_TABLE_DATA_JSON = `
      {
        "cities": [
          {"city": "Seattle", "state": "WA"},
          {"city": "Chicago", "state": "IL"}
        ]
      }`;
```